flixel/FlxCamera.hx

```diff
@@ -713,6 +781,85 @@ class FlxCamera extends FlxBasic
 	}
 	
 	/**
+	 * Helper method preparing debug rectangle for rendering in blit render mode
+	 * @param	rect	rectangle to prepare for rendering
+	 * @return	trasformed rectangle with respect to camera's zoom factor
+	 */
+	@:noCompletion
```

Comment:
I'm not a big fan of `@:noCompletion`, things still show up in API docs with that (see [`startQuadBatch()`](http://api.haxeflixel.com/flixel/FlxCamera.html?#startQuadBatch) for instance). There's `@:dox(hide)`, but I feel like it's cleaner to just make them `private` to begin with (and use `@:access` to access them)?